ghost/core/core/server/api/endpoints/utils/serializers/input/posts.js

```diff
@@ -24,6 +35,33 @@ function removeSourceFormats(frame) {
     }
 }
 
+/**
+ * Selects all allowed columns for the given frame.
+ * 
+ * This removes the lexical and mobiledoc columns from the query. This is a performance improvement as we never intend
+ *  to expose those columns in the content API and they are very large datasets to be passing around and de/serializing.
+ * 
+ * NOTE: This is only intended for the Content API. We need these fields within Admin API responses.
+ *
+ * @param {Object} frame - The frame object.
+ */
+function selectAllAllowedColumns(frame) {
+    if (!frame.options.columns && !frame.options.selectRaw) {
+        // Because we're returning columns directly from the table we need to remove info columns like @@UNIQUE_CONSTRAINTS@@
+        frame.options.selectRaw = _.keys(_.omit(postsSchema, ['lexical','mobiledoc','@@UNIQUE_CONSTRAINTS@@'])).join(',');
+    } else if (frame.options.columns) {
+        frame.options.columns = frame.options.columns.filter((column) => {
+            return !['mobiledoc', 'lexical'].includes(column);
+        });
+    } else if (frame.options.selectRaw) {
```

Comment:
I don't feel strongly about this but I wonder if we should strip these columns out if they're explicitly being asked for in `selectRaw`. I don't think we do this anywhere currently, so this is more of a maintenance concern than anything, but if we have some future use-case where we need to parse the lexical to generate some calculated field, I could see this causing me an hour or two of confusion when I can't get these fields returned no matter what I do.

Not a blocker at all, just a thought